robot_skills/src/robot_skills/robot.py

```diff
@@ -101,6 +119,18 @@ def __init__(self, robot_name="", wait_services=False):
             not_operational_parts = [name for name, part in self.parts.iteritems() if not part.operational]
             rospy.logwarn("Not all hardware operational: {parts}".format(parts=not_operational_parts))
 
+    # Deprecated warning for 'robot.leftArm' to avoid breaking the code.
+    @proprty
```

Comment:
typo?

Reply:
we have 2 deprecated decorators, it seems, at `tue_robocup/robot_smach_states/src/robot_smach_states/util/designators` and at `tue_robocup/robot_skills/src/robot_skills/util/decorators.py`, we may want to unify this at some point.